src/App.tsx

```diff
@@ -7,33 +8,60 @@ function getRandomName(): string {
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+type State = {
+  hasClock: boolean,
+  clockName: string,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    hasClock: true,
+    clockName: getRandomName(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    document.addEventListener('mouseup', this.handleClick);
```

Comment:
Hide and show clock doen't work. Fix it.
<img width="852" alt="Screenshot 2022-09-14 at 11 29 35" src="https://user-images.githubusercontent.com/17098736/190103383-471533f2-ec65-4548-b5d8-a8b4e35cf15f.png">